test/gtest/uct/ib/test_ib.cc

```diff
@@ -954,5 +971,75 @@ UCS_TEST_SKIP_COND_P(test_uct_event_ib, txrx_cq,
     m_e2->flush();
 }
 
-
 UCT_INSTANTIATE_IB_TEST_CASE(test_uct_event_ib);
+
+class test_uct_ib_mtu : public test_uct_ib {
+public:
+    void create_connected_entities() {
+        modify_config("IB_PATH_MTU", "4096");
+        m_e1 = uct_test::create_entity(0);
+        modify_config("IB_PATH_MTU", "2048");
+        m_e2 = uct_test::create_entity(0);
+
+        m_entities.push_back(m_e1);
+        m_entities.push_back(m_e2);
+
+        m_e1->connect(0, *m_e2, 0);
+        m_e2->connect(0, *m_e1, 0);
+    }
+    static ucs_status_t ib_am_bcopy_handler(void *arg, void *data,
+                                            size_t length, unsigned flags) {
+        memcpy(arg, data, length);
+        ++test_uct_ib::m_ib_am_handler_counter;
+        return UCS_OK;
+    }
+
+    static size_t ib_am_bcopy_pack_cb(void *dest, void *arg) {
+        size_t length = (size_t)arg;
+
+        memset(dest, 0, length);
+        return length;
+    }
+
+    void send_recv_bcopy(uct_ep_h ep, entity *ent, size_t length) {
+        size_t start_am_counter = test_uct_ib::m_ib_am_handler_counter;
+        uct_ib_iface_t *iface   = ucs_derived_of(ep->iface, uct_ib_iface_t);
+        std::vector<char> buffer(length);
+        uct_iface_attr_t attr;
+        ucs_status_t status;
+        size_t len;
+
+        check_caps_skip(UCT_IFACE_FLAG_AM_BCOPY);
```

Comment:
can do `check_skip_test()`, because you used `UCS_TEST_SKIP_COND_P`

Reply:
updated